package.json

```diff
@@ -59,5 +59,9 @@
       "<rootDir>/node_modules/react-addons-test-utils",
       "<rootDir>/node_modules/fbjs"
     ]
+  },
+  "dependencies": {
+    "react": "^0.14.7",
```

Comment:
These being here is right, but should we leave them in `devDependencies` too? Lines 43 and 45.